src/main/java/edu/ucsb/cs56/mapache_search/membership/AuthControllerAdvice.java

```diff
@@ -78,7 +106,9 @@ public String getRole(OAuth2AuthenticationToken token) {
     }
 
     private String token2login(OAuth2AuthenticationToken token) {
```

Comment:
maybe the function name should be changed to clarify that it sometimes returns a login, and sometimes an email (unless "login" is an email address? in which case it should be token2email)